### PR TITLE
Add full page video to virtual workflow

### LINKED ISSUE #14642: Move to full-page for convert video to virtual flow
## Description
Move "Convert to Virtual Hearing" functionality from the modal to a full page. Note that most of the disabled states are built in #14421. This issue will focus on moving away from the modal and verifying all AC.

Chandra: add in the remainder of video - virtual details.

## Acceptance criteria
- [ ] Feature toggle: [yes]
- [ ] This feature should be accessible to the following user groups: Hearing Coordinators
- [ ] Include screenshot(s) in the Github issue
- [ ] Modify feature tests

### Detailed AC
([Figma link to start of flow](https://www.figma.com/file/V87TZArfdurCGJiEjQ73ES/Virtual-Hearings?node-id=5649%3A26461))
The major update here is moving "Convert to Virtual Hearing" functionality from the current modal to a full page, in order to maintain consistency with the new Central Office designs. Unlike formerly-Central hearings which now calculate timezone based on veteran address, however, formerly video hearings will continue to calculate timezone based on the Regional Office.
- [ ] The Convert to Virtual Hearing page displays the following info / form fields:
  - [ ] Hearing Date - read-only
  - [ ] Hearing Time - same radio field that appears for Video hearings on the Daily Docket, except that the option values make the Regional Office timezone explicit, i.e. the option values format is [h:mm am/pm RegionalOfficeTimezone / h:mm am/pm BoardEasternTimezone]. Same formatting applies to values in the Other dropdown.
  - [ ] Veteran / Appellant name and mailing address - read-only
  -  [ ] Veteran / Appellant Email with help text
      - required field (no change to current state)
      - pre-populated (no change to current state)
  - [ ] Power of Attorney (POA) type label, name, and address - read-only
      - note the various POA types/labels: Attorney or Service Organization
      - `Open Question: for some Service Organizations, a person's name, in addition to service org name, displays on the Daily Docket but not on Case Details. Is this name being pulled from VACOLS or VBMS? Should we try to display it here?`
      - [ ] if no POA, display message ([direct link to design applied to Central](https://www.figma.com/file/V87TZArfdurCGJiEjQ73ES/Virtual-Hearings?node-id=5649%3A19281))
  - [ ] POA / Representative Email with help text
      - optional
      - pre-populated (no change to current state)
  - [ ] VLJ dropdown
      - optional
  - [ ] VLJ Email - read-only, based on dropdown selection; if blank, display `None`
- [ ] On pressing the Convert to Virtual Hearing button, the async jobs to create the conference and send emails begin, and the user is taken back to Hearing Details where the In Progress and and Success states/alerts display as appropriate
  - No changes to current state alert messaging or form field disabled states

### PR DESCRIPTION
**Pull request by sahalliburton**
Resolves #14642 

### Description
Checks for feature flag to determine whether to use the full page video to virtual conversion or the virtual hearing modal.

### Acceptance Criteria
- [ ] Code compiles correctly

### Testing Plan

**Full Page**
1. Shadow user `BVASYELLOW`
1. Navigate to the hearing schedule and choose a Video hearing for Oakland Regional Office: http://localhost:3000/hearings/schedule
1. Click the 'Edit hearing details' link
1. Change the type to Virtual
1. Ensure the full page mockup matches the Figma diagram  for video-to-virtual
1. Click the Convert to Virtual Hearing Button
1. Change the hearing type back to Video
1. Ensure the full page mockup matches the Figma diagram for virtual-to-video

**Modal**
1. Open a rails console and disable the full page feature: `FeatureToggle.disable!(:full_page_video_to_virtual)`
1. Shadow user `BVASYELLOW`
1. Navigate to the hearing schedule and choose a Video hearing for Oakland Regional Office: http://localhost:3000/hearings/schedule
1. Click the 'Edit hearing details' link
1. Change the type to Virtual
1. Ensure the Modal is shown and not the full page
1. Click the Convert to Virtual Hearing Button
1. Change the hearing type back to Video
1. Ensure the Modal is shown and not the full page

### User Facing Changes
 - [x] Screenshots of UI changes added to PR & Original Issue

 BEFORE|AFTER
 ---|---
ModalVideoToVirtual|FullPageVideoToVirtual
ModalVirtualToVideo|FullPageVirtualToVideo

### CODE REVIEW

**Comment by tomas-nava** [comment]
the spacing & header-size changes look great! I will re-review once #14861 and #14854 are merged into master and then into this branch, to make sure the time bugs are addressed.

File: client/app/hearings/utils.js [hunk 1]
```diff
@@ -267,7 +267,7 @@ export const zoneName = (time, name, format) => {
 export const hearingTimeOptsWithZone = (options, local) =>
   options.map((item) => {
     // Default to using EST for all times before conversion
-    moment.tz.setDefault(zone === true ? 'America/New_York' : zone);
+    moment.tz.setDefault(local === true ? 'America/New_York' : local);
```
Comment (tomas-nava) [inline client/app/hearings/utils.js:270]: good catch

**Comment by nicholer** [comment]
> I have one question about the hearing time drop downs in the daily docket and on the 'convert to' pages; are they supposed to be so verbose?
> Seems like we could at least get rid of the `(US & Canada)` part, but I defer to @nicholer

The time selection on demo (screenshot below) currently only shows one time - the RO time for video dockets; ET for Central dockets. Is it possible to continue doing that? I don't remember if we discussed a specific need to see more than one time displayed in the Board hearing time selection (although the timezone selections do display more than one).

Screen Shot 2020-08-10 at 3 54 17 PM

**Comment by sahalliburton** [comment]
@nicholer in the Figma diagrams it was showing the RO time over the Central office time: https://www.figma.com/file/V87TZArfdurCGJiEjQ73ES/Virtual-Hearings?node-id=5649%3A26461. Should I remove it so that it displays only the RO time? Thanks!

**Comment by nicholer** [comment]
@sahalliburton Thanks for the link/reminder :)

I'm torn on this. On the Daily Docket, showing only the RO time is cleaner and aligns with what Hearing Coordinators are used to seeing. But on the Convert to Virtual page, it _seems_ (I don't have specific user feedback to support this) useful to also show the translation to Board/Central time; if you're looking at the Daily Docket to be sure you're not scheduling for overlapping times, the Daily Docket is ordered by Central/ET time not the RO time.

I'm open to discussing further. But for now I'm leaning _slightly_ toward leaving the dropdown values as is, as wordy as they are. Only because in the next few months we'll be looking at redesigning the Daily Docket for virtual-only days.

**Pull request merged**